pkg/brew/tome.rb

```diff
@@ -0,0 +1,24 @@
+class Tome < Formula
+  HOMEPAGE = "https://github.com/toumorokoshi/tome"
+
+  desc "Modern replacement for 'sub'"
+  homepage HOMEPAGE
+  url HOMEPAGE
+  license "MIT"
+  version '0.1.0'
```

Comment:
in these situations, should the version match the version of the binary released on GitHub?

Reply:
Yes. Which will require keeping version synchronized in two place.

I could break out a task to make a `version_bump` helper script that updates the version in both places 🤔 to make sure it reliably happens?

Reply:
Promoted to new issue.